assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -183,8 +183,7 @@ button {
 }
 
 .blurb {
-  font-size: 14px;
-  line-height: 1.4;
+  font: 100 16px/1.25 $gu-headline;
```

Comment:
what is the unitless `100` font-size?

Reply:
It's the font-weight, sadly browsers don't agree on what the keywords like `lighter` mean exactly. And with variable fonts, it's going to be https://medium.com/@clagnut/get-started-with-variable-fonts-c055fd73ecd7 🤣